simmechanics/CMakeLists.txt

```
@@ -281,6 +281,10 @@ generate_icub_simmechanics(YARP_ROBOT_NAME iCubGazeboV2_5
                            YAML_TEMPLATE "icub2_5/ICUB_2-5_BB_simmechanics_options.yaml.in"
                            CSV_TEMPLATE "icub2_5/ICUB_2-5_BB_joint_parameters.csv.in"
                            INCREASE_INERTIA_FOR_GAZEBO)
+generate_icub_simmechanics(YARP_ROBOT_NAME iCubGazeboV3
```

```suggestion
generate_icub_simmechanics(YARP_ROBOT_NAME iCubV3
```

As in this case we don't do any Gazebo-specific modification, could we call it simply `iCubV3`?

---

For me it is ok, I thought that was the convention `iCubGazeboV..`

---

That was because the `iCubGazeboV*` have a lot of workarounds to run in Gazebo (see https://github.com/robotology/icub-models/issues/33), so if now there are not anymore Gazebo-related workarounds, it could be a good idea to just have an `iCubV3` that is the canonical model for iCub3, that can be loaded in any program that loads URDFs.